dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedMetricsTRSOpenApiIT.java

```diff
@@ -702,4 +737,21 @@ public static List<RunExecution> createRunExecutions(int numberOfExecutions) {
         }
         return executions;
     }
+
+    public List<RunExecution> convertROCrateToRunExecution(String roCratePath) throws IOException {
+        try {
+            List<RunExecution> runExecutions = new ArrayList<>();
+            String roCrate = new String(Files.readAllBytes(Paths.get(roCratePath)));
+            Map<String, Object> map = GSON.fromJson(roCrate, Map.class);
+            RunExecution execution = new RunExecution();
+            execution.setExecutionStatus(RunExecution.ExecutionStatusEnum.SUCCESSFUL);
+            execution.setDateExecuted(Instant.now().toString());
+            execution.setAdditionalProperties(map);
+            runExecutions.add(execution);
+            return runExecutions;
+        } catch (IOException ex) {
+            System.out.println(ex.getMessage());
```

Comment:
It would be better to use the Logger and pass the exception itself as an argument -- that way the stack trace gets logged if we need to figure out what went wrong.